src/otx/config/data/default.yaml

```diff
@@ -4,6 +4,9 @@ data_root: ${base.data_dir}
 mem_cache_size: 1GB
 mem_cache_img_max_size: null
 
+tile_config:
+  enable_tiler: false
+
```

Comment:
Is this mean we can use tiling for all task?

Reply:
Ideally, the implementation of tiling should extend to all tasks. This feature is now available for detection and instance segmentation, as OTX 1.5 introduced support for both tasks. 

Looking ahead, it'd be good for OTX 2.0 to expand tiling to include semantic segmentation and classification tasks, given the use cases (e.g. segmenting/classifying tiny cells on large medical images, etc). 

It's not clear to me if we can use tiling for action tasks, but it doesn't seem like any obvious problems are preventing it from being supported.